src/main/resources/application-local.properties

```diff
@@ -1,8 +1,9 @@
 spring.jpa.hibernate.ddl-auto=update
 spring.datasource.url=jdbc:mysql://127.0.0.1:3306/viz?useSSL=false
-spring.datasource.username=root
-spring.datasource.password=Sept@1994
+spring.datasource.username=yourdbusername
+spring.datasource.password=yourdbpassword 
```

Comment:
I have generalized the credentials. add username and password for your installation settings in FAQ.